typescript/cli/src/commands/core.ts

```diff
@@ -0,0 +1,13 @@
+import { CommandModule } from 'yargs';
+
+import { coreDeploy } from '../deploy/core.js';
+import { log } from '../logger.js';
+
+import { deployWith } from './deploy.js';
+
+export const coreCommand: CommandModule = {
+  command: 'core',
+  describe: 'Core command',
+  builder: (yargs) => yargs.command(deployWith(coreDeploy)),
+  handler: () => log('Command required'),
+};
```

Comment:
There should be child commands here right? Presumably deploy isn't the only thing related to core we'll ever want to to (e.g. read, update)

Reply:
Yup, it will look something like this
```typescript
export const coreCommand: CommandModule = {
  command: 'core',
  describe: 'Core command',
  builder: (yargs) =>
    yargs
    .command(deploy(deployCore))
    .command(read(updateCore))
    .command(update(updateCore))
    .version(false)
    .demandCommand(),
  handler: () => log('Command required'),
};


```